lib/InteractiveViewer.mli

```diff
@@ -0,0 +1,21 @@
+(** This module defines the main user interface and interaction logic for a diff viewer application using the Nottui library. It provides mechanisms to navigate through patches and view them in different formats, respond to user inputs, and run test simulations. *)
+
+val view : Patch.t list -> Nottui.ui Lwd.t
+(** [view patches] constructs a reactive UI for navigating and viewing a list of patches.
+    It supports toggling between help view and normal view, and handling navigation commands.
+    @param patches A list of patches to be displayed and navigated through.
+    @return A reactive UI element constructed using Lwd combinators. *)
+
```

Comment:
This function is internal to this module and is actually an implementation detail. It should not be part of the interface.

For instance, suppose we decide to change the internal implementation of diffcessible and stop using `Nottui`. In this case, this function would change type, making it an unnecessary breaking change.